jcache/embedded/src/main/java/org/infinispan/jcache/embedded/JCache.java

```diff
@@ -534,11 +543,12 @@ public void removeAll(Set<? extends K> keys) {
             // does not block the commit of other, already persisted entries
             AggregateCompletionStage<Void> stage = CompletionStages.aggregateCompletionStage();
             keys.stream()
-                .map(k -> rwMapSkipCacheLoad.eval(k, Remove.getInstance()))
-                .forEach(stage::dependsOn);
+                  .map(k -> rwMapSkipCacheLoad.eval(k, Remove.getInstance()))
+                  .forEach(stage::dependsOn);
             CompletionStages.join(stage.freeze());
          } else {
-            rwMapSkipCacheLoad.evalMany(keys, Remove.getInstance()).forEach(b -> {});
+            rwMapSkipCacheLoad.evalMany(keys, Remove.getInstance()).forEach(b -> {
+            });
```

Comment:
-1

Reply:
Maybe you could check the Intellij style and update it? https://issues.redhat.com/browse/ISPN-11588